examples/metrics/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # ---
 # Stage 1: Install certs, build binary, create default config file
 # ---
-FROM ghcr.io/project-zot/golang:1.19 AS builder
+FROM ghcr.io/project-zot/golang:1.20 AS builder
```

Comment:
Ideally, this should be build-arg'ed out but that also means whoever uses this needs to be aware.